src/extensions/functions/checkFile.js

```diff
@@ -0,0 +1,5 @@
+const fs = require('fs')
+
+module.exports = (rule, context) => {
+  return fs.existsSync(rule.location)
```

Comment:
I used existsSync here because fs.exists is [deprecated](https://nodejs.org/api/fs.html#fs_fs_exists_path_callback).

Reply:
looks great!

There's no way you'd know this unless you were good friends with gluegun, but the context has a filesystem attached to it, that is basically [fs-jetpack](https://github.com/szwacz/fs-jetpack).  So you could do
`const {filesystem} = context`
and then do `filesystem.exists`. which will return "dir" or "file" which is perfect.
https://github.com/szwacz/fs-jetpack#existspath